resources/views/fly/dockerignore.blade.php

```diff
@@ -0,0 +1,28 @@
+# excludes from the docker image/build
+
+# 1. Ignore Laravel-specific files we don't need
+bootstrap/cache/*
+storage/framework/cache/*
+storage/framework/sessions/*
+storage/framework/views/*
+storage/logs/*
+*.env*
+.rr.yml
+rr
+frankenphp
+vendor
```

Comment:
Not suggesting any change; but in the future, this may be something that is tailored based on options and detected features.  Example: if you are not using frankenphp, no need to include it.  Here's an example from Rails:

https://github.com/fly-apps/dockerfile-rails/blob/main/lib/generators/templates/dockerignore.erb

And a smaller one from JS:

https://github.com/fly-apps/dockerfile-node/blob/main/templates/.dockerignore.ejs

Reply:
True! Gonna add this as a card to the project